golos.vesting/parameters.hpp

```diff
@@ -41,29 +41,30 @@ struct vesting_delegation : parameter {
         eosio::check(return_time > 0, "delegation return_time <= 0");
     }
 };
-using delegation_param = param_wrapper<vesting_delegation,5>;
+using vesting_delegation = param_wrapper<vesting_delegation_t,5>;
 
-struct bwprovider: parameter {
-    permission_level provider;
+struct vesting_bwprovider_t: parameter {
+    name actor;
+    name permission;
```

Comment:
why did you expand it? `permission_level` is more clear than 2 separate fields

Reply:
@zxcat Because in original ABI they are expanded, parameter used not only in action but also in table. Can we change it?

Reply:
I see. ok, let them stay expanded as you wrote